src/applications/debt-letters/components/DebtLettersTable.jsx

```diff
@@ -94,8 +95,10 @@ export const DebtLettersTable = ({ debtLinks }) => {
                 />
                 <span aria-hidden="true">Download letter </span>
                 <span className="sr-only">
-                  Download {debtLetter.typeDescription} dated{' '}
-                  {moment(debtLetter.receivedAt).format('MMM D, YYYY')}
+                  Download {debtLetter.typeDescription} dated
+                  <span className="vads-u-margin-left--0p5">
+                    {moment(debtLetter.receivedAt).format('MMM D, YYYY')}
```

Comment:
I usually use `LL` for formatting... spelling out the month is better for a11y

Reply:
Made this [ticket](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/department-of-veterans-affairs/va.gov-team/25717) for our design team to look at.